src/sentry/models/groupsnooze.py

```diff
@@ -60,6 +61,8 @@ class Meta:
 
     __repr__ = sane_repr("group_id")
 
+    USER_COUNT_CACHE_TIMEOUT = 300
```

Comment:
Can you add a comment to here and explaining the unit of this variable? Is it `ms`?